sources/packages/backend/apps/api/src/app.institutions.module.ts

```diff
@@ -43,14 +43,18 @@ import {
   EducationProgramOfferingControllerService,
 } from "./route-controllers";
 import { AuthModule } from "./auth/auth.module";
-import { EducationProgramOfferingValidationService } from "./services/education-program-offering/education-program-offering-validation.service";
-import { WorkflowClientService, SequenceControlService } from "@sims/services";
+import { SequenceControlService, WorkflowClientService } from "@sims/services";
+import {
+  MSFAANumberService,
+  IntegrationStudentRestrictionService,
+} from "@sims/integrations/services";
 import {
   SFASIndividualService,
   SFASApplicationService,
   SFASPartTimeApplicationsService,
 } from "@sims/services/sfas";
 
+
```

Comment:
Please remove the extra blank line.
@ann-aot on my VS Code this is resolving when saving the file. Is it not working for you?

Reply:
ya... it's working... I had merge conflicts in these files. and I was on a call with dheepak to resolve that. and I think it made prettier slow, which could cause this. fixing all files